pkg/operator/ceph/controller/spec.go

```diff
@@ -601,6 +603,40 @@ fi`,
 	}
 }
 
+// GenerateReadinessProbeExecDaemon generates a readiness probe that makes sure a daemon is ready
+// to process traffic, and that it returns 0
+func GenerateReadinessProbeExecDaemon(daemonType, daemonID string) *v1.Probe {
+	confDaemon := getDaemonConfig(daemonType, daemonID)
+	command, msg := confDaemon.buildDaemonReadinessCheckCommand()
+	return &v1.Probe{
```

Comment:
This is the same probe as the liveness probe, right? If so, let's implement it similar to the `GenerateStartupProbeExecDaemon()`, which first calls the liveness probe method, then just overrides some parameters. Then it removes the code duplication.

Reply:
Actually, I don't see how the liveness and readiness probes could be the same. Is there a difference I'm missing? The liveness probe wouldn't typically fail on the standby, while we always expect the readiness probe to fail on the standby.

Reply:
The commands used in liveness(mon_status) and readines(mgr_status) probe are different, and what we want to expose in the the readiness probe is just that the standby mgr service is ok but it is not ready to process traffic.

Reply:
Ok I see now where the command is different, just the `command` var is different and formatted into the overall action. 
@rkachach Could you just factor out a helper method to avoid duplication of the ProbeHandler?

Reply:
Fixed